src/lib/core/say.h

```diff
@@ -163,7 +170,14 @@ typedef int (*log_format_func_t)(struct log *log, char *buf, int len, int level,
 struct log {
 	/** The current file descriptor. */
 	int fd;
+	/**
+	 * Indicates whether the current file is a terminal (tty).
+	 * This field is non-zero if the file descriptor refers to a terminal.
+	 */
+	int isatty;
 	/** The current log level. */
+	/** This field is non-zero if STDERR_FILENO is a terminal (tty). */
+	int isatty_stderr;
 	int level;
```

Comment:
I suggest to rename the first field a `isatty_stdin` for consistency.

Reply:
replaced